app/Models/VenueFacility.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+class VenueFacility extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'venue_id',
+        'facility_id',
```

Comment:
remove extra coma

Reply:
trailing commas is best practice isn't it? ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas & https://stackoverflow.com/questions/2829581/why-do-php-array-examples-leave-a-trailing-comma 🤣️ kebiasaan prettier js